client/auth/signup.tsx

```diff
@@ -260,8 +260,8 @@ export function Signup() {
         required(t('auth.passwordValidator.confirm', 'Confirm your password')),
         matchesOther('password', t('auth.passwordValidator.matching', 'Enter a matching password')),
       ),
-      ageConfirmation: requireChecked(t('common.literals.required', 'Required')),
-      policyAgreement: requireChecked(t('common.literals.required', 'Required')),
+      ageConfirmation: requireChecked(t('auth.ageConfirmation.required', 'Required')),
```

Comment:
Would probably be good to use the same string for all `required/requireChecked` validators if possible (similar to min/maxLength)